Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Destination/UpdateDestination.cs

```diff
@@ -59,8 +60,18 @@ public static class UpdateDestination
                 }
                 else
                 {
-                    document = await createNewDocument(data, log, collection);
-                    result = new CreatedResult("", document.ToJson());
+                    string type = Convert.ToString(data.location.type);
+
+                    if (removeDestination(type, data))
```

Comment:
I don't understand something..
This if statement will occur when the value of the document is null, that's means that there's no document for this location. How can you remove something that doesn't exists?

Reply:
Please explain better what you do not understand.